rest/bulk_api.go

```diff
@@ -276,9 +285,10 @@ func (h *handler) handleDump() error {
 func (h *handler) handleRepair() error {
 	// TODO: If repair is re-enabled, it may need to be modified to support xattrs and GSI
 	err := errors.New("_repair endpoint disabled")
-	if err != nil {
+	/*if err != nil {
```

Comment:
This looks like a change in auditing behaviour (to not audit when a no-op call to repair is made).  Does it make sense in this PR?

Reply:
golangci-lint ci didn't pass without this change, I am not immediately sure why because it's not changed code.

Reply:
Now it _did_ pass with this removed, I fixed this issue on main but anemone hasn't been rebased recently.